public/app/plugins/datasource/grafana-azure-monitor-datasource/resourcePicker/resourcePickerData.ts

```diff
@@ -0,0 +1,173 @@
+import { FetchResponse, getBackendSrv } from '@grafana/runtime';
+import { getLogAnalyticsResourcePickerApiRoute } from '../api/routes';
+import { EntryType, Row, RowGroup } from '../components/ResourcePicker/types';
+import { AzureResourceSummaryItem } from '../types';
+import { SUPPORTED_LOCATIONS, SUPPORTED_RESOURCE_TYPES } from './supportedResources';
+
+const RESOURCE_GRAPH_URL = '/providers/Microsoft.ResourceGraph/resources?api-version=2020-04-01-preview';
+
+interface RawAzureResourceGroupItem {
+  subscriptionId: string;
+  subscriptionName: string;
+  resourceGroup: string;
+  resourceGroupId: string;
+}
+
+interface RawAzureResourceItem {
+  id: string;
+  name: string;
+  subscriptionId: string;
+  resourceGroup: string;
+  type: string;
+  location: string;
+}
+
+interface AzureGraphResponse<T = unknown> {
```

Comment:
I think this is the same as whatever the default is so I'm not sure this is useful anymore?

Reply:
This object describes the actually JSON that comes from ARG. The full response from fetch will be something like

```
Promise<
  {
    ok: boolean;
    data: {
      data: RawAzureResourceGroupItem[]
    }
  }
>
```

This type with the interface represents the inner most object with `data`

Reply:
ahhh the double data tripped me up thanks!